build.gradle

```diff
@@ -130,6 +130,24 @@ subprojects {
         input = files('src/main/kotlin', 'src/test/kotlin', 'src/funTest/kotlin')
     }
 
+    dokka {
+        externalDocumentationLink {
+            url = new URL('https://codehaus-plexus.github.io/plexus-containers/plexus-container-default/apidocs/')
+        }
+
+        externalDocumentationLink {
+            url = new URL('https://fasterxml.github.io/jackson-databind/javadoc/2.9/')
```

Comment:
Instead of hard-coding `2.9`, should we use something like `jacksonVersion.tokenize('.').subList(0, 2).join('.')`?

Reply:
I thought about it, too, but I let it be to keep it simple.